src/HttpPacket.h

```diff
@@ -0,0 +1,23 @@
+/**
+ * Project MyHttpServer
+ * @author Pei-Ru Wang, Cheryl Huang, Yuan Zhou
+ * @version 0.0.1
+ */
+
+
+#ifndef _HTTPPACKET_H
+#define _HTTPPACKET_H
+
+#include <unordered_map>
+#include <string>
+
+// Encapsulate http packets.
+class HttpPacket {
+public: 
+    std::unordered_map<std::string, std::string> header;
```

Comment:
Is this class copyable?

Reply:
Please add variable comment.
https://google.github.io/styleguide/cppguide.html#Variable_Comments